### DISCUSSION TITLE
Page showing up as raw markdown instead of HTML

### DISCUSSION THREAD

**Discussion by hyeokhyen** [greenelab/lab-website-template#222] [Q&A] [answered]
Hi,

I plan to create '.md' files for each course in the teaching page. For example, "/your-lab-website/teaching/fall-2018/index.md". 
However, when I compiled locally it only shows the raw md file, instead of compiled HTML. Please let me know how to resolve this.

**Comment by vincerubinetti** [comment, marked as answer]
Do you have a front matter in the file? See the last note on this page:

https://greene-lab.gitbook.io/lab-website-template-docs/basics/edit-pages

If it's not that, you'll need to share your repo or provide much more details.

**Reply by hyeokhyen** [reply]
Thanks a lot! I got it to work. The front matter was the key :)
